Packs/SIGNL4/Integrations/SIGNL4/SIGNL4.yml

```diff
@@ -0,0 +1,53 @@
+category: Messaging and Conferencing
+commonfields:
+  id: SIGNL4
+  version: -1
+configuration:
+- display: Team or Integration Secret
+  name: secret
+  hiddenusername: true
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+description: SIGNL4, mobile alerting.
```

Comment:
I recommend more detail.
![alt text](image.png)

Reply:
Will to in the next commit. Thanks.